Make sure to include config.h so that NEEDS_* macros are properly
taken into account. This was a problem for NEEDS_YYWRAP, which was set
to 1 in config.h, but the corresponding code wasn't compiled in.

Index: at-3.1.13/parsetime.l
===================================================================
--- at-3.1.13.orig/parsetime.l
+++ at-3.1.13/parsetime.l
@@ -4,6 +4,7 @@
 #include <time.h>
 #include "y.tab.h"
 #include "parsetime.h"
+#include "config.h"
 
 char *last_token = NULL;
 char **my_argv;
